### PR TITLE
Warmboot script improvements - timeout exec, disable swss autorestart, remove trap

### PR DESCRIPTION
**Pull request by vaibhavhd**
#### What I did
Related PR for 201811 branch - https://github.com/Azure/sonic-utilities/pull/1474

Below changes are made to warmboot/fastboot script:

1. Disable swss auto-restart during warm reboot.
2. Add timeout to make sure syncd shutdown request will return in time. 5s
3. Disable trap handler after +e.
4. Make sure that syncd pre-shutdown wait won't take more than 60 seconds.
5. Make sure subsequent docker exec won't stuck for long time
6. Before shutdown, check docker exec on the relevant docker containers still works.

#### How to verify it
Tested warmreboot on a physical testbed.

### CODE REVIEW

**Review by jleveque — APPROVED** [review]

**Comment by vaibhavhd** [comment]
retest this please

File: scripts/fast-reboot [hunk 1]
```diff
@@ -715,6 +739,9 @@ if [ -x ${DEVPATH}/${PLATFORM}/${PLATFORM_PLUGIN} ]; then
     ${DEVPATH}/${PLATFORM}/${PLATFORM_PLUGIN}
 fi
 
+# Re-enable swss autorestart for the next boot
+config feature autorestart swss enabled
```
Comment (vaibhavhd) [inline scripts/fast-reboot:743]: @jleveque in this step my intention was to re-enable autorestart of `swss`. However, in my tests, I have noticed that after warm-reboot `swss autorestart` remains disabled.
Do you know what am I doing missing here?

Comment (jleveque) [reply scripts/fast-reboot:743]: I can only assume the saved configuration has it marked as disabled? But I see you're not saving the config after disabling it, so it's not you doing it. But I would start by checking that, and if so, figure out why it's saved as disabled.

Comment (vaibhavhd) [reply scripts/fast-reboot]: I checked again, and this was due to database container being killed/stopped before that, therefore it was not able to enable it back
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/redis/connection.py", line 559, in connect
    sock = self._connect()
  File "/usr/local/lib/python3.7/dist-packages/redis/connection.py", line 615, in _connect
    raise err
  File "/usr/local/lib/python3.7/dist-packages/redis/connection.py", line 603, in _connect
    sock.connect(socket_address)
ConnectionRefusedError: [Errno 111] Connection refused
```

Further, I found out that after executing `systemctl stop swss`, the container do not automatically restart even if autorestart is enabled for swss at the time.
In comparison, for `201811` branch, after `docker kill swss`  monit/systemctl timers keeps restarting swss

**Review by yxieca — APPROVED** [review]

**Pull request merged**